apps/web/src/utils/helpers.ts

```diff
@@ -269,3 +269,10 @@ export function unpackOptionalArray<T = []>(
   }
   return array
 }
+
+// Markdown is impossible to detect in all cases, but this should cover most of the cases we'll run into
```

Comment:
Don't think we need this check `ProposalDescription` and the `rehypeRaw, rehypeSanitize and remarkGfm` plugins in the `ReackMarkdown` -- it handles both markdown and html